src/components/internal/DocsTable/styled.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+import { em } from 'polished';
+
+import { Table } from '../../Table';
+
+export const Styled = styled(Table)`
+  tr td:first-child {
+    width: 20%;
+  }
+
+  tr td:nth-child(2) {
+    width: 20%;
+  }
+
+  tr td:nth-child(3) {
+    width: 15%;
+  }
+
+  code {
+    font-family: monospace;
+    padding: ${({ theme }) => em(theme.honeycomb.size.micro)}
+      ${({ theme }) => em(theme.honeycomb.size.tiny)};
+    margin: 0 ${({ theme }) => em(theme.honeycomb.size.micro)};
+    border: 1px solid ${({ theme }) => theme.honeycomb.color.border};
+    border-radius: ${({ theme }) => em(theme.honeycomb.radius.reduced)};
+    background-color: ${({ theme }) => theme.honeycomb.color.bg.input.normal};
+    color: ${({ theme }) => theme.honeycomb.color.text.masked};
+  }
+`;
+
+export const Asterisk = styled.span`
+  ::before {
+    content: ' *';
```

Comment:
There is a space before asterisk - is the space here intended?

Reply:
<img width="78" alt="image" src="https://user-images.githubusercontent.com/15721063/103751347-05915500-506d-11eb-99c5-84b7faea7c86.png">

Yes - I think it looks a tiny bit better. 😂

Reply:
Got it! In general I tend to use `padding-left` to control this spacing. Let me know what you think or if it is not the case here

Reply:
Totally fair, it looks more readable. Change done!